testng.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name = "Suite">
```

Comment:
The name of Suite is Suite :)
Remember about the Names convention. Other developers read code as well :)